src/app/view/redButton..js

```diff
@@ -1,6 +1,6 @@
 export default function ButtonRed(text) {
```

Comment:
The button should probably receive some kind of a function that shuold be attached to it as a listener, that would react on click. What should happen when you click on the button?